test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetRestoreCommandTest.cs

```diff
@@ -3337,6 +3337,61 @@ public void RestoreCommand_WithProjectWithPackagesConfig_DefaultSettings_Package
             r.AllOutput.Should().Contain($"Package 'packageA' 1.1.0 has a known high severity vulnerability");
         }
 
+        [SkipMono()]
+        public void InstallCommand_WithPackageWithVulnerabilities_RaisesWarnings()
```

Comment:
Install tests go in a different file: https://github.com/NuGet/NuGet.Client/blob/dev/test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetInstallCommandTest.cs